src/app/pages/client/explore/Server.tsx

```diff
@@ -299,17 +301,27 @@ function LimitButton({ limit, onLimitChange }: LimitButtonProps) {
                   <Chip variant="SurfaceVariant" onClick={() => setLimit('96')} radii="Pill">
                     <Text size="T200">96</Text>
                   </Chip>
+                  <Chip variant="SurfaceVariant" onClick={() => setLimit('10000')} radii="Pill">
+                    <Text size="T200">Max</Text>
+                  </Chip>
```

Comment:
It won't be a good idea to show this option considering that the performance will be poor.